src/components/common/Form/index.js

```diff
@@ -69,9 +69,11 @@ export class Form extends Component {
 
   render() {
     const childWithProp = React.Children.map(this.props.children, child => {
-      return React.cloneElement(child, {
-        validateForm: this.handleCheckInputValid,
-      });
+      if (child.type !== 'div') {
```

Comment:
This seems like a bit of a code smell - why is this necessary?

Reply:
I got a feeling the solution then is to dig into why the validation doesn't like these rules and fix it from there. Gonna take a closer look at this PR now.

Reply:
Can you not return `null` instead of empty divs?